todoman/model.py

```diff
@@ -252,6 +252,27 @@ def complete(self) -> None:
         self.percent_complete = 100
         self.status = "COMPLETED"
 
+    def undo(self) -> Todo | None:
+        """
+        Immediately restores this todo. Marks it as needs action, resets the
+        percentage to 0 and deletes the completed_at datetime.
+
```

Comment:
```suggestion
        """Immediately restores this todo.

        Marks it as needs action, resets the percentage to 0 and deletes the
        completed_at datetime.
```

The first line is show in short help output.